lib/core/jtrace.c

```diff
@@ -550,6 +742,275 @@ j_trace_enter(gchar const* name, gchar const* format, ...)
 		G_UNLOCK(j_trace_echo);
 	}
 
+	if (j_trace_flags & J_TRACE_ACCESS)
+	{
+		if (strcmp(name, "ping") == 0)
+		{
+			va_list args;
+			va_start(args, format);
+			g_free(trace_thread->client.program_name);
+			trace_thread->client.program_name = strdup(va_arg(args, const char*));
+			trace_thread->client.uid = va_arg(args, guint32);
+			va_end(args);
+		}
+		else if (strncmp(name, "backend_", 8) == 0)
```

Comment:
maybe add a comment here to clarify that this line filters for trace calls with `backend_` prefix.